invenio_rdm_records/theme/__init__.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+# Copyright (C) 2020 Northwestern University.
+#
+# Invenio RDM Records is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio RDM Records."""
+
+from __future__ import absolute_import, print_function
```

Comment:
We can skip these guys now, as we're no longer supporting Python 2

Reply:
True, removed all.